### DISCUSSION TITLE
openwrt编译后，一直无法安装核心，请问怎么直接把核心一同编译进去？

### DISCUSSION THREAD

**Discussion by YunHair** [UnblockNeteaseMusic/luci-app-unblockneteasemusic#312] [Q&A]
### 需求描述

openwrt编译后，一直无法安装核心，请问怎么直接把核心一同编译进去？

### 预期行为

预编译核心

### OpenWrt 版本

23

**Comment by 1715173329** [comment]
https://github.com/UnblockNeteaseMusic/luci-app-unblockneteasemusic/blob/js/root/usr/share/unblockneteasemusic/update.sh#L47

**Reply by hhht110** [reply]
天灵大佬也在啊？ 你还是用的23 啊？  404大佬已经换了内核了 换成6.6.73了。。也是核心永远无法安装。。难过呢。。。我刚给提交饭 才看到你 哈哈
